ledger/participant-state/kvutils/src/main/protobuf/daml_kvutils.proto

```diff
@@ -357,6 +360,9 @@ message DamlSubmissionDedupKey {
 
   // Uploading participant's id.
   string participant_id = 3;
+
+  // Seed use for interpretation.
+  bytes submissionSeed = 4;
```

Comment:
Why is submission seed here? I don't see how this can work, since we must also be able to deduplicate commands that are sent by different participants (we support a deployment where we have clones of the participant for high availability).

Reply:
It has nothing to do here. I dropped it. Good catch